main/src/ca/uwaterloo/flix/language/phase/Lexer.scala

```diff
@@ -17,12 +17,14 @@ package ca.uwaterloo.flix.language.phase
 
 import ca.uwaterloo.flix.api.Flix
 import ca.uwaterloo.flix.language.CompilationMessage
-import ca.uwaterloo.flix.language.ast.{Ast, ReadAst, Token}
+import ca.uwaterloo.flix.language.ast.{Ast, LexerErr, ReadAst, Token, TokenKind}
 import ca.uwaterloo.flix.util.{ParOps, Validation}
 import ca.uwaterloo.flix.util.Validation._
-
 import scala.collection.mutable
 
+
+// TODO: internals? something to handle $DEFAULT$
```

Comment:
We have built-in "functions". They are always written `$FOO$`.

Reply:
Random example: https://github.com/flix/flix/blob/master/main/src/library/Int32.flix#L307